docker/ltcd/start-ltcctl.sh

```diff
@@ -43,8 +43,12 @@ RPCUSER=$(set_default "$RPCUSER" "devuser")
 RPCPASS=$(set_default "$RPCPASS" "devpass")
 NETWORK=$(set_default "$NETWORK" "simnet")
 
-exec ltcctl \
-    "--$NETWORK" \
+PARAMS=""
+if [ "$NETWORK" != "mainnet" ]; then
+    PARAMS=$(echo --$NETWORK)
+fi
+
+PARAMS=$(echo $PARAMS \
```

Comment:
This breaks the whole command.
1. The `exec ltcctl` part is gone
2. Missing closing parens
3. Why assigning to `PARAMS` again?

Reply:
Ups, thanks for finding that!